sql/updates/world/3.3.5/9999_99_99_99.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM `spell_script_names` WHERE `spell_id` = -980;
```

Comment:
always delete by ScriptName xD

Reply:
Still think it's a terrible idea, and still haven't found this design document you guys never link.

Reply:
1 spell can have multiple scripts.

Reply:
and many spells can share the same script, which is more common.

Reply:
When you delete by spell_id, you will delete all scripts assigned to it... and sometimes it's not what you want...
We delete By ScriptName, because if you are adding that Script, you will always deleted only script that you changing/adding

Reply:
I prefer delete by ScriptName and spell_id 😄

Reply:
There are a total of 17 spell ids with multiple ScriptNames, compared to 253 spell ids which share ScriptNames.
And of those 17 I see "clever" solutions which are unnecessary and come from developer laziness for the most part (hello, those paladin seals).
Those 17 are special exception cases, do not let them dictate a very unhygienic practice.

Reply:
If you want to keep your sanity in databases you refer to objects by their unique explicit identifier, which in this case is the spell id given by Blizzard, not the arbitrary and mutable name given by TrinityCore.

Reply:
Well maybe @Shauren can explain better.